CHANGES.rst

```diff
@@ -1,4 +1,4 @@
-3.0 (unreleased)
+﻿3.0 (unreleased)
```

Comment:
I'm not sure why that line had changed, either - I double checked for extra white spaces there but found none.